Initial cut at saving JIMS1058

colbywhite opened the pull request:
This just saves the file in the expected spot. As noted in the TODO list in the `README`, next would be to commit the change.

Should we close the original open-austin/project-ideas#73 and start bringing the convo to this repo?

colbywhite committed:
initial ruby skeleton commit

colbywhite committed:
add logic to save today's JIMS1058

colbywhite committed:
update readme with basic info

luqmaan commented:
How do you plan to to deploy this?

I'd prefer we use something thats compatible with AWS Lambda (Node.js, Python and Java). That way we don't have to setup a server with cron. https://aws.amazon.com/lambda/

Take a look at https://github.com/open-austin/construction-permits as an example.

colbywhite commented:
How in the world did I not realize that Lambda doesn't support ruby? :doh: Well, python it is.

luqmaan commented:
😘 Yeah it sucks they don't support ruby.

luqmaan commented on .gitignore:
```diff
@@ -0,0 +1,2 @@
+*.swp
+.DS_Store
```
Need a newline here

colbywhite replied:
FYI: this has been fixed in 7a782aa. The GitHub UI didn't like it when I rebased that one data file out.

luqmaan commented on .rubocop.yml:
```diff
@@ -0,0 +1,3 @@
+# IntelliJ Default
+Metrics/LineLength:
+  Max: 120
```
Newline, although I guess this file will be going away.

colbywhite committed:
remove ruby files

how did i not know that AWS lambda did not have ruby support?
:disappointed:

colbywhite committed:
reimplement into python.

same behavior, different language

colbywhite committed:
placeholder for data files; add to TODO

will not start filling the dir up until an agreement has been reached
regarding privacy concerns

also need to do tests

colbywhite commented:
@luqmaan, It's been switched to python. Enjoy. 😉 

@fileunderjeff, In response to your comment about privacy on open-austin/project-ideas#73, I rebased the one data file I had in here out. I won't start saving data until Open Austin comes to a consensus on how they want to handle it.
